js/lib/altcha/altcha.min.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Rather than committing 3rd-party code directly to our repository, is it possible to install this via an NPM dependency or, failing that, possibly even with Composer?